### PR TITLE
dcos-2528 param for list_to_err is [str] not [Error]

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: dcos/util.py [hunk 1]
```diff
@@ -387,7 +387,7 @@ def list_to_err(errs):
     """convert list of error strings to a single string
 
     :param errors: list of string errors
-    :type errors: list of strings
```
Comment (mgummelt) [inline dcos/util.py]: s/errors/errs

**Comment by mgummelt** [comment]
LGTM

File: dcos/util.py (hunk 1 above)
Comment (tamarrow) [inline dcos/util.py]: done

**Comment by tamarrow** [comment]
thanks

**Pull request merged**